react_frontend/src/components/Home.js

```diff
@@ -1,20 +1,85 @@
-import React from 'react'
-import '../assets/NavBar.css'
+import React, { useState, useEffect } from 'react'
+import '../assets/Home.css'
 import SearchBar from './SearchBar'
 import SearchButton from './SearchButton'
```

Comment:
I also wanted to ask what is the react select in Home.js? I also wanted to address that you should try to remove unnecessary imports that you do not need.

Reply:
Good catch! I'll go ahead and fix that 👍🏾